packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -654,17 +654,27 @@ export const RemoteServerFactory = {
      * This function assumes that optionally, there is a `setup-server.js`
      * file in each extension directory in the build.
      *
-     * This file should export a default which is a function.
+     * This file should export a default which is a IApplicationExtension class.
      *
      * @private
      */
     _setupExtensions(app, options) {
+        logger.info('Setting up extensions...')
+
+        // TODO: support extensions options array syntax i.e. ['extension-a', {}]
         const extensions = options.mobify?.app?.extensions || []
+        logger.info('Extensions to load', {
+            namespace: 'DevServerMixin._setupExtensions',
+            additionalProperties: {extensions: extensions}
+        })
+
         app.__extensions = extensions || []
```

Comment:
Minor: `extensions` should already be defined on line 665 (with at least an empty array).

```suggestion
        app.__extensions = extensions
```